coramin/third_party/tests/test_minlplib_tools.py

```diff
@@ -0,0 +1,143 @@
+import coramin
+import unittest
+import os
+
+
+class TestMINLPLibTools(unittest.TestCase):
+    def test_get_minlplib_instancedata(self):
+        current_dir = os.path.abspath(os.path.dirname(__file__))
+        coramin.third_party.get_minlplib_instancedata(download_dir=os.path.join(current_dir, 'minlplib'))
+        self.assertTrue(os.path.exists(os.path.join(current_dir, 'minlplib', 'instancedata.csv')))
+        os.remove(os.path.join(current_dir, 'minlplib', 'instancedata.csv'))
+        os.rmdir(os.path.join(current_dir, 'minlplib'))
+
+    def test_filter_minlplib_instances(self):
+        current_dir = os.path.abspath(os.path.dirname(__file__))
+
+        cases = coramin.third_party.filter_minlplib_instances(instancedata_download_dir=os.path.join(current_dir, 'minlplib'),
```

Comment:
There is no guarantee that the `test_get_minlplib_instancedata` test will run before this test.  In fact, it should never (nosetests tends to alphabitize).

Reply:
Fixed.